test/TestUtilities/Test.Utility/Test.Utility.csproj

```diff
@@ -3,9 +3,9 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <TargetFrameworks>$(TargetFrameworksExe)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksExe);netcoreapp3.0</TargetFrameworks>
```

Comment:
Update the property in the cmmon.targets

Reply:
Thanks! Changed accordingly.